pages/home.html

```diff
@@ -100,10 +106,8 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    <em>Built by the community for the community</em> – Oppia aims to enable
```

Comment:
The inversion here feels a bit weird to me, too. Is the italicized text meant to be a headline?

What is the purpose of the change?